src/main/java/org/folio/template/util/BarcodeImageGenerator.java

```diff
@@ -0,0 +1,58 @@
+package org.folio.template.util;
+
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import org.apache.commons.lang3.StringUtils;
+import org.krysalis.barcode4j.impl.code128.Code128Bean;
+import org.krysalis.barcode4j.output.bitmap.BitmapCanvasProvider;
+import org.krysalis.barcode4j.tools.UnitConv;
+
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.Base64;
+
+public class BarcodeImageGenerator {
+  private static final Logger LOG = LoggerFactory.getLogger("mod-template-engine");
+  private static final String MIME_TYPE_PNG = "image/x-png";
```

Comment:
Is it standard mime type?
According to https://www.iana.org/assignments/media-types/media-types.xhtml#image
it's image/png

image/png
--




image/png
--




image/png
--

Reply:
They are the same thing. And `Barcode4J`, the library I used to generate images, supports both. PNG used to be a non-standard MIME type up until 1996, hence the `x-` prefix :)

Reply:
fixed